content/22.vaccines.md

```diff
@@ -255,9 +258,81 @@ Some common successful whole virus vaccine include targeting influenza viruses,
+In the phase I trial, the overall incidence of adverse reactions was 29-38% of patients in the 0 to 14 day group and 13-17% in the 0 to 28 day vaccination group.
+The most common symptom was pain at the injection site, which was reported by 17-21% of patients in the 0 to 14 day cohort and 13% in the 0 to 28 day cohort.
+Most adverse reactions were mild (grade 1) where patients recovered within 48 hours.
+There was a single case of acute hypersensitivity with manifestation of urticaria 48 hours following the first dose of study drug was reported in the 6 &mu;g group
+Most adverse reactions were mild (grade 1) in severity and participants recovered within 48 h. 
+There was a single case, from the 6 &mu;g group, of acute hypersensitivity with manifestation of urticaria 48 h after the first dose.
+Both the 14-day and 28-day cohorts had a strong neutralizing Ab response.
+
+In a study with older patients (older than 60 years), they performed another single-center, randomized, double-blind, placebo-controlled, phase I/II trial [@10/fx8z].
+The phase I trial looked at dose escalation using 3 dosages: 1.5, 3 and 6 &mu;g.
+The mean age of participants was 65.8 years (std = 4.8).
+Of 95 screened participants, 72 were enrolled.
+These 72 participants were split into low (3 &mu;g) and high (6 &mu;g) dose groups.
+Within each groups 24 were treated and 12 were placebo.
+A neutralizing antibody response against live SARS-CoV-2 was detected compared to baseline.
+This response was similar across the two dose concentrations.
+Additionally, they did not observe a difference in response between age groups (60–64 years, 65–69 years, and ≥70 years).
+
+In the phase II the mean age was 66.6 years (std = 4.7).
+499 participants were screened and 350 were enrolled.
+300 were evenly split into 1.5, 3 and 6 &mu;g dose groups.
+The remaining 50 were assigned to the placebo group.
+Again, they found a neutralizing antibody response in phase II.
+There wasn't a significant different between the response using 3 &mu;g versus 6 &mu;g, but the response was higher compared to using 1.5 &mu;g.
+
+Participants were required to record adverse reaction events within the first 7 days after each dose.
+The safety results were combined across phage I and II.
+All adverse reactions were either mild (grade 1) or moderate (grade 2) in severity.
+The most common symptom was pain at the injection site (9%) and fever (3%).
+2% (7 participants) reported severe adverse events (4 from the 1.5 &mu;g group, 1 from the 3 &mu;g group, 2 from the 6 &mu;g group), though these were found to be unrelated to the vaccine.
+
+
+The phase III trials are currently being conducted in Brazil [@doi:10.1186/s13063-020-04775-4].
+This is a randomized, multicenter, endpoint driven, double-blind, placebo-controlled clinical trial.
+They are expecting 13,060 participants with 11,800 ages 18 to 59 years and 1,260 age 60+.
+Participants will be health care professionals.
+
```

Comment:
I would suggest possibly turning this last paragraph into sort of a summary paragraph, where you add a summary sentence or two just to say where things are at as of present and then say that the phase III trial is ongoing